src/Apps/Order/Routes/Shipping2/Utils/computeOrderData.ts

```diff
@@ -40,10 +40,10 @@ type SavedShippingQuoteData = {
   shippingQuotes: Array<{ id: string; isSelected: boolean }>
 } | null
 
-export const parseOrderData = (
+export const computeOrderData = (
   order: ShippingProps["order"],
   me: ShippingProps["me"]
-): ParsedOrderData => {
+): ComputedOrderData => {
```

Comment:
non-blocking - Again not a huge sticking point for me, but just for context the naming changed a few times and i settled on _parsed_ to match what we were calling it in volt.

Reply:
Yah i felt kinda weird changing it as its pretty trivial, but it came down to
- Thinking of "parsing" as taking input data, and reformatting _the exact same data_ into another shape for output
- While "computed" data -- think, [computed functions](https://easy-peasy.vercel.app/docs/api/computed.html), which is a concept -- takes inputs and returns different outputs, as we're doing here